packages/common/src/data/App/index.json

```diff
@@ -4,19 +4,19 @@
       "id": 1,
       "icon": "flaticon-atom",
       "title": "Discover processes",
-      "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor. "
+      "description": "Browse the processes & find all tools you need to get started and elevate your operations."
     },
     {
       "id": 2,
       "icon": "flaticon-trophy",
-      "title": "Vote on the best",
-      "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor."
+      "title": "Vote for the best",
+      "description": "Share your review and experiences, take a coin and place your vote on the workflows you approve."
```

Comment:
Share your review and experience. Vote for your best processes to support the community.